lstchain/visualization/plot_dl2.py

```diff
@@ -49,7 +49,7 @@ def plot_features(data, true_hadroness=False):
              histtype=u'step',bins=100,
              label="Protons")
     plt.ylabel(r'# of events',fontsize=15)
-    plt.xlabel(r"$log_{10}E$(GeV)")
+    plt.xlabel(r"$log_{10}E$[TeV]")
```

Comment:
Should be `r'$\log_{10}(E_\mathrm{true} / TeV)$'`

Reply:
done